CMakeLists.txt

```diff
@@ -3,10 +3,11 @@
 #
 #   cd build
 #   cmake ../SLiM
-#   make
+#   make -j10
```

Comment:
This doesn't need to be a part of the standard build procedure now, does it?  On some people's machines (like my laptop) this will make the build slower, since fewer than ten cores exist, right?  Unless I'm missing something, I'd suggest leaving this to individual discretion rather than making it a part of the standard build recipe.

Reply:
Depends on whether you like waiting longer. Using multiple threads (however many you wish, 10 is an example, I actually use 60-80 on my servers) typically speeds up things immensely. Most Linux people are well aware of this, and can change the parameter value according to their machine.

Reply:
Hmm.  SLiM has a lot of novice users who are biologists, not developers, and have no idea what -j does.  (I had to look it up myself, since I don't typically build at the command line.)  They won't know how to tailor this, and on their desktop machines with only a couple of cores, -j10 would make the build slower, no?  Or does -j just express a max, and on a machine with fewer cores it would spawn fewer than 10 threads?  As you say, experienced Linux users will know to use -j anyway, so there's probably no need to suggest it; and the inexperienced may suffer a penalty from this suggestion, no?